cmd/gateway-gcs.go

```diff
@@ -942,7 +942,13 @@ func (l *gcsGateway) CompleteMultipartUpload(bucket string, key string, uploadID
 
 	var parts []*storage.ObjectHandle
 	for _, uploadedPart := range uploadedParts {
-		parts = append(parts, l.client.Bucket(bucket).Object(gcsMultipartDataName(uploadID, uploadedPart.ETag)))
+		parts = append(parts, l.client.Bucket(bucket).Object(gcsMultipartDataName(uploadID,
+			uploadedPart.PartNumber, uploadedPart.ETag)))
+	}
+
+	// Returns name of the composed object.
+	gcsMultipartComposeName := func(uploadID string, composeNumber int) string {
+		return fmt.Sprintf("%s/tmp/%s/composed-object-%05d", gcsMinioMultipartPathV1, uploadID, composeNumber)
```

Comment:
I just wanted to confirm that `minio.sys.temp/multipart/v1/tmp/UPLOADID/` is directory than `minio.sys.temp/tmp/...`